docs/operate-a-stake-pool/node-installation-process.md

```diff
@@ -216,18 +216,18 @@ We can now build the `Haskell-based` `cardano-node` to produce executable binari
 cabal build cardano-node cardano-cli
 ```
 
-Install the newly built node and CLI commands to the $HOME/.local/bin directory:
+Install the newly built node and CLI commands to the $HOME/local/bin directory:
```

Comment:
I sincerely believe this change is very wrong since `~/.local` has become a Linux fixture, especially with KDE and GNOME, while ever since the early days of UNIX `~/local` would in fact be discouraged... it only appears without a dot in often locally writable directories like `/usr/local`.

Reply:
fixed in https://github.com/cardano-foundation/developer-portal/pull/1162/commits/8b0f3f1d503b67e7701e116f20397a7857ab0b77